genrl/deep/common/buffers.py

```diff
@@ -3,9 +3,81 @@
 import random
 import numpy as np
 from typing import Tuple
+from .utils import get_obs_shape, get_action_dim
 
 
 class ReplayBuffer:
+    def __init__(self, size, env):
```

Comment:
Docs for Replay Buffer need to be added. We can do that somewhere else since this seems holding up other PRs

Reply:
Yeah. Raised an issue for that. I'll work on that. #154